pkg/kvstore/allocator/allocator.go

```diff
@@ -438,12 +440,40 @@ func (k *kvstoreBackend) RunGC(ctx context.Context, rateLimit *rate.Limiter, sta
 
 	staleKeys := map[string]uint64{}
 
+	min := uint64(minID)
+	max := uint64(maxID)
+	reasonOutOfRange := fmt.Sprintf("out of local cluster identity range [%d,%d]", min, max)
+
 	// iterate over /id/
 	for key, v := range allocated {
 		// if k.lockless {
 		// FIXME: Add DeleteOnZeroCount support
 		// }
 
+		// Parse identity ID
+		items := strings.Split(key, "/")
+		if len(items) == 0 {
+			log.WithField(fieldKey, key).WithError(err).Warning("Identity GC: unknown identity key, skip it")
```

Comment:
Nit: typically, the subsystem (`logfields.LogSubsys`) will categorize the log msgs, so there's no need to prefix the log msgs here with "Identity GC"

```suggestion
			log.WithField(fieldKey, key).WithError(err).Warning("Unknown identity key found, skipping")
```